integration/v3_grpc_test.go

```diff
@@ -193,3 +195,225 @@ func TestV3DeleteRange(t *testing.T) {
 		clus.Terminate(t)
 	}
 }
+
+// TestV3WatchFromCurrentRevision tests Watch APIs from current revision.
+func TestV3WatchFromCurrentRevision(t *testing.T) {
+	tests := []struct {
+		initialKeys   []string
```

Comment:
I feel we should call this putKeys rather than initialKeys. We first do the watch, not initialize key space.